src/Board.elm

```diff
@@ -0,0 +1,25 @@
+module Board exposing (..)
+
+import List exposing (repeat, take, drop, head)
+
+type alias Board = List String
```

Comment:
Why using `List` as the data structure here if you really want to `set`(you call it `mark`) a value in a specific index and `get`(you call it `markAt`) a value in a specific index?

Is there another data structure that simplifies this operations?